infra/spawn_pipeline.py

```diff
@@ -128,9 +131,11 @@ def main():
         for task_id, task_payload in tasks:
             queue.createTask(task_id, task_payload)
 
-        print(f"https://community-tc.services.mozilla.com/tasks/groups/{task_group_id}")
+        logger.info(
+            "https://community-tc.services.mozilla.com/tasks/groups/%s", task_group_id
+        )
     except taskcluster.exceptions.TaskclusterAuthFailure as e:
-        print(f"TaskclusterAuthFailure: {e.body}", file=sys.stderr)
+        logger.info("TaskclusterAuthFailure: %s", e.body)
```

Comment:
```suggestion
        logger.exception("Failed to authenticate with Taskcluster")
```
Using `exception` could be a better fit than `info` in this case.